### PR TITLE
update ticket received log line to be summary of tickets

### PR DESCRIPTION
**Pull request by ad-astra-video**
**What does this pull request do? Explain your changes. (required)**
Moves `Receiving ticket` log line to be summary log line of tickets processed.  Was a debug log line that printed for each ticket processed.

**Specific updates (required)**
- one file updated

**How did you test each of these updates (required)**
@papabear99 ran similar version on mainnet 

**Does this pull request close any open issues?**
No

**Checklist:**

- [ ] Read the [contribution guide](./CONTRIBUTING.md)
- [X] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

**Review by thomshutt — APPROVED** [review]

File: core/orchestrator.go [hunk 1]
```diff
@@ -198,6 +198,8 @@ func (orch *orchestrator) ProcessPayment(ctx context.Context, payment net.Paymen
 			ev := ticket.EV()
 			orch.node.Balances.Credit(sender, manifestID, ev)
 			totalEV.Add(totalEV, ev)
+			totalFaceValue.Add(totalFaceValue, ticket.FaceValue)
+			totalWinProb.Add(totalWinProb, ticket.WinProbRat())
```
Comment (leszko) [inline core/orchestrator.go:202]: 1. So, we never set those to `0` again and the log will always show the cumulative tickets?
2. I wonder how much sense it takes to add the probabilities and face values together? What this information actually may give you?

**Review by leszko — COMMENTED** [review]
@ad-astra-video Added a comment. Thanks for the PR.

File: core/orchestrator.go (hunk 1 above)
Comment (ad-astra-video) [reply core/orchestrator.go:202]: 1. These variables are initialized in the ProcessPayment function so start at 0 every time. The log line is intended to show the cumulative amounts for the tickets (face value/win prob/ev).   
2. To me, this is a debug log line so more info is likely better right?

Comment (leszko) [reply core/orchestrator.go:202]: Ahh, you're right. I missed that part that they are initialized for each payment processing. Thansk for the explanation!

**Review by leszko — APPROVED** [review]

**Comment by stronk-dev** [comment]
Shouldn't `winProb` be the average win prob rather than the sum?

**Comment by ad-astra-video** [comment]
> Shouldn't `winProb` be the average win prob rather than the sum?

I don't think the average gets you to the `faceValue` mathematically.  Also, wouldn't the average be the `winProb` of one ticket in the batch? The way I looked at it was ten tickets at 10 gwei gets 100 gwei in tickets with a cumulative `winProb` of .000008333.  100 gwei / .000008333 / 1,000,000 = .012 ETH.

**Comment by stronk-dev** [comment]
What does the sum of `winProb` or `faceValue` represent? These are per ticket params
If you earn 10 tickets of 0.12FV and 10% win percentage, that's not equivalent to 1 ticket of 1.2FV and 100% win percentage. Only the total EV makes sense to sum up for a summary, ie:
```
Payment tickets processed 10 tickets with average faceValue=1.2, winProb=10%, ev=0.12 for a total EV of 1.2
```

**Comment by ad-astra-video** [comment]
OK, see now. I was approaching from the reverse starting with EV / total win prob.  I have no issue removing the winProb and faceValue totals. Was just trying to reduce the spam in logs from ticket processing on debug log level. Will get PR in at some point.

**Comment by stronk-dev** [comment]
i mean it's good to reduce logging, i just brought it up cause the change breaks this panel of mine (since there's no way to translate the sums back to a per-ticket basis with the new log format):
image

**Pull request merged**